custom_components/tahoma/light.py

```diff
@@ -44,19 +44,16 @@ def __init__(self, tahoma_device, controller):
         self._skip_update = False
         self._effect = None
 
-        device_type = self.tahoma_device.widget
         self.update()
 
     @property
     def brightness(self) -> int:
         """Return the brightness of this light between 0..255."""
-        _LOGGER.debug("[THM] Called to get brightness %s" % (self._brightness))
         return int(self._brightness * (255 / 100))
```

Comment:
doesn't self._brightness need to be defined in the __init__ method?

Reply:
Not necessarily, since it will be set in the update method. (which is called via the constructor). However, added it for clarity and to be sure.

Not sure if the update() function should be called from init.

Reply:
I'd say no... I haven't seen this in other components.

Reply:
Removing it breaks the component, so I will keep it for now.

Reply:
strange...
but ok! merge it :)